spark/dl/src/main/scala/com/intel/analytics/bigdl/example/mkldnnint8/TestImageNet.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.example.mkldnnint8
+
+import com.intel.analytics.bigdl.models.resnet.ImageNetDataSet
+import com.intel.analytics.bigdl.nn.Module
+import com.intel.analytics.bigdl.optim._
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric._
+import com.intel.analytics.bigdl.utils._
+import org.apache.log4j.{Level, Logger}
+import org.apache.spark.SparkContext
+
+/**
+ * This example is to evaluate trained resnet50 with imagenet data and get top1 and top5 accuracy
```

Comment:
This example demonstrates how to evaluate pre-trained resnet-50 with ImageNet dataset using Int8